grammar.js

```diff
@@ -489,13 +492,48 @@ module.exports = grammar({
     // https://stackoverflow.com/questions/13014947/regex-to-match-a-c-style-multiline-comment
     marginalia: _ => seq('/*', /[^*]*\*+(?:[^/*][^*]*\*+)*/, '/' ),
 
-    compound_statement: $ => seq(
```

Comment:
In nvim-treesitter is a indentens.scm which refenrences `compound_Statement` we should have added a copy of this to our repo as we so that the tests fail here when we rename nodes.